tbx/static_src/sass/components/_motif-heading.scss

```diff
@@ -23,6 +23,12 @@
         #{$root}--static & {
             animation: none;
         }
+
+        &--i {
+            background-size: 200%;
+            animation: motifAnimationForLeterI 1.7s cubic-bezier(0.3, 0, 0.6, 1)
+                forwards;
```

Comment:
I think because this class is more specific, it isn't getting the preferred-reduce-motion rule. Adding the below inside the --i modifier should sort it:

```
@include reduced-motion() {
    animation: none;
}
```

Reply:
Added in 277bf97bcaaca8f7c85a4e87ffc35f1ba3afb549